app/knights/knight_prep.py

```diff
@@ -0,0 +1,18 @@
+from app.knights.create_knight import CreateKnight
+list_of_actions = ["weapon", "armour", "potion"]
+
+
+def knight_preparation(

```

Comment:
Rename this function to `prepare_knights` (name with a verb that represents action, not gerund), filename to `support_functions.py` or `utils.py`.